examples/js/loaders/GLTFLoader.js

```diff
@@ -565,17 +565,21 @@ THREE.GLTFLoader = ( function () {
 
 			getMaterialType: function () {
 
-				return THREE.ShaderMaterial;
+				return 'KHR_MATERIALS_PBR_SPECULAR_GLOSSINESS'; //best thing would probably be a sub class of StandardMaterial?
 
 			},
 
 			extendParams: function ( params, material, parser ) {
 
 				var pbrSpecularGlossiness = material.extensions[ this.name ];
 
-				var shader = THREE.ShaderLib[ 'standard' ];
-
-				var uniforms = THREE.UniformsUtils.clone( shader.uniforms );
+				// these are only the extra uniforms, keep the rest hidden under StandardMaterial
+				var uniforms = {
+					specular: { value: new THREE.Color().setHex( 0x111111 ) },
+					glossiness: { value: 0.5 },
+					specularMap: { value: null },
+					glossinessMap: { value: null }
+				};
```

Comment:
I realize this isn't new with your changes, but let's use the defaults from the extension spec:

specular: 0xffffff
glossiness: 1.0

https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness